boolean/test_boolean.py

```diff
@@ -760,6 +760,28 @@ def test_simplify(self):
         )
         assert result.pretty() == expected.pretty()
 
+    def test_absorption_invariant_to_order(self):
+        algebra = BooleanAlgebra()
+
+        a, b = algebra.symbols(*"ab")
```

Comment:
could you simply create tow symbols in a more readable way, one on each line?

Reply:
Or even on one line, but not as a quirky string deconstruction:
```suggestion
        a, b = algebra.symbols("a", "b")
```

(The original saves all of _three_ characters, in the weirdest way I've ever seen.)